ShopperInsights/src/test/java/com/braintreepayments/api/ShopperInsightsClientUnitTest.kt

```diff
@@ -24,91 +20,30 @@ import org.junit.Test
 
-    private val context: Context = mockk(relaxed = true)
-    private val applicationContext: Context = mockk(relaxed = true)
     private lateinit var sut: ShopperInsightsClient
     private lateinit var api: ShopperInsightsApi
     private lateinit var braintreeClient: BraintreeClient
-    private lateinit var deviceInspector: DeviceInspector
 
     @Before
     fun beforeEach() {
         api = mockk(relaxed = true)
         braintreeClient = mockk(relaxed = true)
-        deviceInspector = mockk(relaxed = true)
-        every { context.applicationContext } returns applicationContext
-        sut = ShopperInsightsClient(api, braintreeClient, deviceInspector)
+        sut = ShopperInsightsClient(api, braintreeClient)
     }
 
-    /**
-     * Tests if the getRecommendedPaymentMethods method returns paypal and venmo recommendations
-     * when providing a shopping insight request.
-     */
     @Test
-    fun testGetRecommendedPaymentMethods_noInstalledApps_returnsDefaultRecommendations() {
-        every { deviceInspector.isVenmoInstalled(applicationContext) } returns false
-        every { deviceInspector.isPayPalInstalled(applicationContext) } returns false
-
-        val request = ShopperInsightsRequest("fake-email", null)
-        sut.getRecommendedPaymentMethods(context, request) { result ->
-            assertNotNull(result)
-            val successResult = assertIs<ShopperInsightsResult.Success>(result)
-            assertFalse(successResult.response.isPayPalRecommended)
-            assertFalse(successResult.response.isVenmoRecommended)
-        }
-        verifyStartedAnalyticsEvent()
-        verifySuccessAnalyticsEvent()
-    }
+    fun `when getRecommendedPaymentMethods is called, started event is sent`() {
+        sut.getRecommendedPaymentMethods(mockk(relaxed = true), mockk(relaxed = true))
 
-    @Test
-    fun testGetRecommendedPaymentMethods_oneInstalledApp_returnsDefaultRecommendations() {
-        every { deviceInspector.isVenmoInstalled(applicationContext) } returns true
-        every { deviceInspector.isPayPalInstalled(applicationContext) } returns false
-
-        val request = ShopperInsightsRequest("fake-email", null)
-        sut.getRecommendedPaymentMethods(context, request) { result ->
-            assertNotNull(result)
-            val successResult = assertIs<ShopperInsightsResult.Success>(result)
-            assertFalse(successResult.response.isPayPalRecommended)
-            assertFalse(successResult.response.isVenmoRecommended)
-        }
         verifyStartedAnalyticsEvent()
-        verifySuccessAnalyticsEvent()
     }
 
     @Test
-    fun testGetRecommendedPaymentMethods_hasBothAppsInstalled_returnsSuccessResult() {
-        every { deviceInspector.isVenmoInstalled(applicationContext) } returns true
-        every { deviceInspector.isPayPalInstalled(applicationContext) } returns true
+    fun `when getRecommendedPaymentMethods is called, succeeded event is sent`() {
```

Comment:
Looks like the test name here and on line 247 got mixed up?

Reply:
Shoot, good catch! let me update this.